docs/topics/pages.md

```diff
@@ -427,7 +427,7 @@ This will then make `related_links` available as a relation across all page type
 
 ## Working with pages
 
-Wagtail uses Django's [multi-table inheritance](https://docs.djangoproject.com/en/stable/topics/db/models/#multi-table-inheritance) feature to allow multiple page models to be used in the same tree.
+Wagtail uses Django's [multi-table inheritance](inv:django#meta-and-multi-table-inheritance) feature to allow multiple page models to be used in the same tree.
```

Comment:
Here

Reply:
PR up https://github.com/wagtail/wagtail/pull/11495

Reply:
Yep, this is one of the ones that works, but the i18n one for example will not support this.

From my re-reading of the docs, it tries to be smart and find any reference that can match, so in theory this could link to the non-models page if the same ref gets added there in Django docs. They may have similar checks to ours to block duplicate refs but not sure.

It seems to be a bit of a try what you can and run the CI.

This is why our CI needs to flag these errors - so it's easy to know if broken links exist.